pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/Worker.java

```diff
@@ -82,7 +83,11 @@ protected void start() throws Exception {
         }
     }
 
-    private static URI initialize(WorkerConfig workerConfig)
+    public static void setEnableInitialDlogMetadataByPulsarTool(boolean enable) {
```

Comment:
How is this method used?

Reply:
User can call `Woker. setEnableInitialDlogMetadataByPulsarTool(true)` or `Woker.setEnableInitialDlogMetadataByPulsarTool(false)`.